src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -20,91 +20,147 @@
 
+<link href="<c:url value="/css/feedback.css"/>" type="text/css" rel="stylesheet"/>
+
 <script src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>" type="text/javascript"></script>
+<script src="/ResourceServingWebapp/rs/jquery-migrate/1.2.1/jquery-migrate-1.2.1.min.js" type="text/javascript"></script>
 <script src="<c:url value="/js/twitlimit-0.2.0.compressed.js"/>" type="text/javascript"></script>
 
-<link href="<c:url value="/css/feedback.css"/>" type="text/css" rel="stylesheet"/>
-
 <c:set var="n"><portlet:namespace/></c:set>
 
 <portlet:actionURL var="postUrl"/>
 
-<h1><spring:message code="feedback.form.question"/></h1>
+<div class="feedback-portlet awesome-bootstrap-checkbox bootstrap-styles">
+<h3 style="font-weight: bold;"><spring:message code="feedback.form.question"/></h3>
 
-<form:form action="${postUrl}" modelAttribute="submitFeedbackForm">       
+<form:form action="${postUrl}" modelAttribute="submitFeedbackForm">
     <spring:bind path="prefs.*">
-    	<c:if test="${status.error}">
-    	<div id="${n}error-message" class="error-message portlet-msg-error portlet-msg error" role="alert" style="display:none">
-   	 	<p><form:errors path="feedback"/></p>
-		</div>
-	    </c:if>
-	</spring:bind>
-	<br>
-	<p id="${n}answer">
-      	<form:radiobutton id="yes" path="like" value="YES"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.yes"/></label>
-      	<form:radiobutton id="no" path="like" value="NO"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.no"/></label>
-      	<form:radiobutton id="maybe" path="like" value="MAYBE"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.maybe"/></label>
-    </p>
-
- 	<p> 		
- 		<label class="portlet-form-field-label"><spring:message code="feedback.form.suggestion"/></label>
- 		<textarea id="${n}feedback" path="feedback" name="feedback" rows="${feedbackRows}" style="width:${feedbackWidth}"></textarea> 		
- 	</p>
- 	<div id="${n}limit" style="margin-bottom: 12px;"></div>
-
- 	<p>
-        <form:checkbox path="anonymous" value="true"/>
-		<label class="portlet-form-field-label"><spring:message code="feedback.form.anonymous"/></label>
-	</p>
-
- 	<input id="${n}useragentstring" type="hidden" name="useragent"/>
- 	<input id="${n}feedbacktabname" type="hidden" name="tabname"/>
-
-    <p>
-       <button type="submit" id="submit" class="feedback-submit-button" disabled="disabled"><spring:message code="feedback.form.submit"/></button>
-    </p>
+        <c:if test="${status.error}">
+        <div id="${n}error-message" class="alert alert-danger" role="alert" style="display:none">
+            <p><form:errors path="feedback"/></p>
+        </div>
+        </c:if>
+    </spring:bind>
+
+    <div data-role="fieldcontain">
+    <fieldset data-role="controlgroup" id="${n}answer">
+        <div class="radio radio-success">
```

Comment:
In the pre-Bootstrap version of the portlet, the radio buttons are inline. Any thoughts on adding radio-inline [1] to the radio buttons?

[1] http://getbootstrap.com/css/#inline-checkboxes-and-radios